### PR TITLE
Update us.m3u

### LINKED ISSUE #12730: Broken: CNN
### Broken Link

https://16live00.akamaized.net/CNN/index.m3u8

### What happened to the stream?

Not loading

### Notes (optional)

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#report-a-broken-stream)

### LINKED ISSUE #12731: Broken: TNT East
### Broken Link

https://16live00.akamaized.net/TNT/index.m3u8

### What happened to the stream?

Not loading

### Notes (optional)

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#report-a-broken-stream)

### PR DESCRIPTION
**Pull request by PopeyeTheSai10r**
- Fixes #12730
- Fixes #12731

### CODE REVIEW

File: streams/us.m3u [hunk 1]
```diff
@@ -787,8 +787,8 @@ https://a.jsrdn.com/broadcast/e6bdcb5ae9/+0000/c.m3u8
 https://d155hi8td9k2ns.cloudfront.net/out/wapo-medialive3-rtmp/live.m3u8
 #EXTINF:-1 tvg-id="ThisTVNetwork.us",This TV Network (480p)
 https://cdn.igocast.com/channel11_hls/channel11_master.m3u8
-#EXTINF:-1 tvg-id="TNTEast.us",TNT
-https://16live00.akamaized.net/TNT/index.m3u8
+#EXTINF:-1 tvg-id="TNTEast.us",TNT [Geo-blocked]
+https://tve-live-ctl.warnermediacdn.com/hls/live/2023168/tnteast/slate/VIDEO_0_3564000.m3u8
```
Comment (Alstruit) [inline streams/us.m3u:791]:
```suggestion
```
Duplicate.

**Review by Alstruit — DISMISSED** [review]

File: streams/us.m3u [hunk 2]
```diff
@@ -249,7 +249,7 @@ https://live.cnbcindonesia.com/livecnbc/smil:cnbctv.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="CNCTV.us",County News Center (CNC TV) (San Diego CA) (720p)
 https://cdn3.wowza.com/5/V2Y2VmhqMEFDTUkx/sdcounty/G0283_012/playlist.m3u8
 #EXTINF:-1 tvg-id="CNN.us",CNN
-https://16live00.akamaized.net/CNN/index.m3u8
+https://tve-live-lln.warnermediacdn.com/hls/live/586495/cnngo/cnn_slate/VIDEO_0_3564000.m3u8
```
Comment (freearhey) [inline streams/us.m3u:252]: Offline.
image

**Review by freearhey — CHANGES_REQUESTED** [review]

File: streams/us.m3u (hunk 2 above)
Comment (freearhey) [reply streams/us.m3u:252]:
```suggestion
https://tve-live-ctl.warnermediacdn.com/hls/live/586495/cnngo/cnn_slate/VIDEO_0_3564000.m3u8
```

File: streams/us.m3u [hunk 3]
```diff
@@ -249,7 +249,7 @@ https://live.cnbcindonesia.com/livecnbc/smil:cnbctv.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="CNCTV.us",County News Center (CNC TV) (San Diego CA) (720p)
 https://cdn3.wowza.com/5/V2Y2VmhqMEFDTUkx/sdcounty/G0283_012/playlist.m3u8
 #EXTINF:-1 tvg-id="CNN.us",CNN
```
Comment (freearhey) [inline streams/us.m3u:251]:
```suggestion
#EXTINF:-1 tvg-id="CNN.us",CNN [Geo-blocked]
```

**Review by freearhey — APPROVED** [review]

File: streams/us.m3u [hunk 4]
```diff
@@ -248,8 +248,8 @@ http://ott-cdn.ucom.am/s65/index.m3u8
 https://live.cnbcindonesia.com/livecnbc/smil:cnbctv.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="CNCTV.us",County News Center (CNC TV) (San Diego CA) (720p)
 https://cdn3.wowza.com/5/V2Y2VmhqMEFDTUkx/sdcounty/G0283_012/playlist.m3u8
-#EXTINF:-1 tvg-id="CNN.us",CNN
-https://16live00.akamaized.net/CNN/index.m3u8
+#EXTINF:-1 tvg-id="CNN.us",CNN [Geo-blocked]
+https://tve-live-ctl.warnermediacdn.com/hls/live/586495/cnngo/cnn_slate/VIDEO_0_3564000.m3u8
```
Comment (freearhey) [inline streams/us.m3u:252]: Now doesn't work either.
image

**Review by freearhey — CHANGES_REQUESTED** [review]

File: streams/us.m3u (hunk 4 above)
Comment (kaikyoofers) [reply streams/us.m3u:252]: Did you try it via USA IP? It is geoblocked

File: streams/us.m3u (hunk 2 above)
Comment (kaikyoofers) [reply streams/us.m3u]: Did you try it via USA IP? It is geoblocked

File: streams/us.m3u (hunk 4 above)
Comment (freearhey) [reply streams/us.m3u:252]: Yes

**Pull request closed without merging**